inc/based_config.php

```diff
@@ -59,6 +59,7 @@
         'GLPI_CONFIG_DIR'      => GLPI_ROOT . '/config', // Path for configuration files (db, security key, ...)
         'GLPI_VAR_DIR'         => GLPI_ROOT . '/files',  // Path for all files
         'GLPI_MARKETPLACE_DIR' => GLPI_ROOT . '/marketplace', // Path for marketplace plugins
+        'GLPI_RESSOURCE_DIR'   => GLPI_ROOT . '/src/Ressources', // Path for marketplace plugins
```

Comment:
This directory should not be customizable. There is no need to declare a constant for it.

Reply:
done